ci/test_challenge.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# This test launch all commands, submit a invalid block and challenge
+
+set -exuo pipefail
+
+CI_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/tmp/ganache.out 2>&1 && pwd )"
+ROOT_DIR=$CI_DIR/..
+
+cd $ROOT_DIR/environment
+yarn
+node index.js clean
+if [ -n "${LOCAL_CORE_SRC+x}" ]; then
+  node index.js prepare --core-src "$LOCAL_CORE_SRC"
+else
+  node index.js prepare
+fi
+node index.js start near-node
+node index.js start ganache
+# Wait for the local node to start
+while ! curl localhost:3030; do
+  sleep 1
+done
+
+while ! curl localhost:9545; do
+  sleep 1
+done
+
+node index.js init-near-contracts
+node index.js init-eth-ed25519
+# Use short lockup time for tests
+node index.js init-near2eth-client --near2eth-client-lock-eth-amount 1e18 --near2eth-client-lock-duration 60
+node index.js init-near2eth-prover
+node index.js init-eth-erc20
+node index.js init-eth-locker
+node index.js init-near-fun-token
+# First start pm2 daemon
+yarn run pm2 ping
+sleep 5
+yarn run pm2 list
+node index.js start near-relay --eth-master-sk 0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501201
+sleep 5
+yarn run pm2 list
+node index.js start eth-relay
+sleep 5
+yarn run pm2 list
+node index.js start near-watchdog
+sleep 5
+yarn run pm2 list
+
+sleep 30
```

Comment:
Awesome, I see you had to increase the bond time to reduce flakiness.

Reply:
Yeah, 10s lock time was too short to send the challenge